GliaWidgets/Sources/View/Chat/Message/GVAPersistentButtonView.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+
+final class GvaPersistentButtonView: OperatorChatMessageView {
+    var onOptionTapped: ((GvaOption) -> Void)!
+
+    private let kLayoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+    private let kImageHeight: CGFloat = 200.0
```

Comment:
I'd propose to stop using ObjC style and don't add `k` prefix for constants. 
@gersonnoboa  @igorkravchenko @EgorovEI  what do you think?

Reply:
fixed [6bf7cd5](https://github.com/salemove/ios-sdk-widgets/pull/683/commits/6bf7cd5b4a195ff9f90c3d6ed2b0b02b7d8ee1ec)

Reply:
There's also the `kImageHeight`

Reply:
fixed [6c89146](https://github.com/salemove/ios-sdk-widgets/pull/683/commits/6c891462d2cdde524421081bad06b1286b0d6d2f)